src/script/components/report-card.ts

```diff
@@ -75,50 +233,84 @@ export class ReportCard extends LitElement {
 
         <div id="report-content">
           <fast-accordion>
-            <fast-accordion-item>
+            <fast-accordion-item
+              @click="${(ev: Event) => this.opened(ev.target)}"
+            >
               <div class="accordion-heading-block" slot="heading">
                 <span class="accordion-heading">Manifest</span>
 
                 <div class="score-block">
-                  <span class="accordion-score">00/40</span>
+                  <span class="accordion-score">${this.maniScore} / 80</span>
```

Comment:
Can we have the max value as a variable instead of hard-coded in the HTML?

Reply:
resolved